SU-SWS/decanter-web
# Add su- prefix content

# READY FOR REVIEW/NOT READY
Ready

# Summary
Adds The `su-` prefix page content. I thought breaking up the second paragraph and adding the bolds helped the text be easier to scan – happy to change back to exactly what is in the Google doc if you'd like. I also wrapped all the classes in `<code>` elements.

# Needed By (Date)
N/A

# Urgency
Not urgent

# Steps to Test
Go to `/for-developers/strategies-and-conventions/the-su-prefix` and ensure the content matches [the Google doc](https://docs.google.com/document/d/1-HNdgVy0ME1zlXdbmr6O9wzGPDEmoxdta8fUHS_0r-c/edit?usp=sharing).

# Affected Projects or Products
N/A

# Associated Issues and/or People
N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
